submissions/dikun/frogger-game/app.js

```diff
@@ -0,0 +1,123 @@
+"use srtict";
+
+const cell = {
+	width: 100,
+	height: 85
+};
```

Comment:
By convention, constants are named all caps and snake-case (e.g. `CELL`, `GAME_FIELD`). This makes them outstand in the code and let the reader instantly know they are constants and not just global variables.